<?xml version="1.0"?>
<launch>

	<!-- Launch Universal Robot ROS Driver. Thsi will also load the robot_description-->
	<!--include file="$(find fcc_ur_launch)/launch/fcc_UR10.launch"/-->

	<!-- Launch barebones rviz-->

	<node name="rviz" pkg="rviz" type="rviz" respawn="false" output="screen"/>
	
	<param name="static_transforms" textfile="$(find scene_manipulation_service)/static_transforms.json" />
	<param name="active_transforms" textfile="$(find scene_manipulation_service)/active_transforms.json" />
	<node name="example_smc_server" pkg="scene_manipulation_service" type="example_smc_server" respawn="true" output="screen"/>

	<node name="example_smc_client" pkg="scene_manipulation_service" type="example_smc_client" respawn="false" output="screen"/>
</launch>
